specification/ai/data-plane/VoiceLive/operations.tsp

```diff
@@ -0,0 +1,15 @@
+import "./models.tsp";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Versioning;
+
+namespace VoiceLive;
+
+// Service versioning is enum based
+#suppress "@azure-tools/typespec-azure-core/no-enum"
```

Comment:
is this the service even using versioning, and please add message to why every single of the suppression is needed

Reply:
The service Uri is /voice-agent/realtime?&api-version=2025-10-01&model=gpt-4o

Reply:
Use it where?

The service definition in websocket.tsp doesn't have an operation, and I'm not sure what we'd add for one. And it does have @versioned used.